server/src/shared/database/seeders/20221110024001-step-ingredients.js

```diff
@@ -1,7 +1,8 @@
 const TABLE_NAME = 'step_ingredients';
 
-const createStepIngredient = (unitOfMeasurementId, ingredientId, stepId) => {
+const createStepIngredient = (unitOfMeasurementId, amount, ingredientId, stepId) => {
```

Comment:
Nitpick: if you are consuming the function arguments in the object in the order "amount -> unitOfMeasurementId -> etc" then they should be in the same order as function arguments 🤓

Reply:
I meant the other way round, you want the function arguments to be in the order of the consumed object 😄